scripts/branch-deployment.sh

```diff
@@ -24,7 +24,7 @@ git clone https://$GITHUB_TOKEN@github.com/axa-ch-webhub-cloud/plib-feature.git
 
 rm -rf ./dist/$BRANCH_NAME/
 mkdir -p ./dist/$BRANCH_NAME
-mv -v ./storybook-static/* ./dist/$BRANCH_NAME/
+cp -R -v ./storybook-static/* ./dist/$BRANCH_NAME/
```

Comment:
Why is this better/solving a previous problem? You may want to comment the shell script with hash comments line-by-line....

Reply:
Instead of mv something and moving back use copy

Reply:
I see, makes sense!